routes.js

```diff
@@ -0,0 +1,94 @@
+const express = require('express');
+const routes = express.Router();
+const Db = require('./data/db.js');
```

Comment:
why capitalise the 'D'. why not just 'db' '. Db'  adds one extra key press, every time you need to do a db action.